src/storage/DataAccessorBasedStore.ts

```diff
@@ -348,6 +348,13 @@ export class DataAccessorBasedStore implements ResourceStore {
     return types.some((type): boolean => type.value === LDP.Container || type.value === LDP.BasicContainer);
   }
 
+  /**
+   * Verifies if this is the metadata of a storage container.
+   */
+  protected isStorage(metadata: RepresentationMetadata): boolean {
```

Comment:
```suggestion
  protected isRootStorage(metadata: RepresentationMetadata): boolean {
```

for a better hint to the spec?